src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -68,6 +68,7 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
   private intervalId: any;
   private updateInterval: number = 10000;//time represents in ms how fast tree updates changes from mainframe
   @ViewChild('fileExplorerUSSInput') fileExplorerUSSInput: ElementRef;
+  private noResultMsg: String;
```

Comment:
We should probably use the primitive type 'string' unless you have a good reason not to